### ISSUE TITLE
Suggestion: Regarding Ignoring players

### ISSUE THREAD

**Issue by GoonaHead** [Cockatrice/Cockatrice#2428]
I find myself "Ignoring" players for multiple reasons, but I see no way of categorizing them. I would like to able to know my reasons for ignoring each one, so I can consider whether to unblock at a later date. 
I would like to see the option to give reasons for blocking people. This might include:
- Rude
- Frequently AFK
- Cheater
- Slow Play
- Leaving drafts early

**Comment by skwerlman** [comment]
being able to set general notes per player would be a neat feature, and would cover this use case

**Comment by Daenyth** [comment]
I agree - I think "notes" would cover this just fine without requiring any changes specific to ignoring. Closing this in favor of #1957
